tpp-app/src/home/components/CycleCard.js

```diff
@@ -51,9 +51,10 @@ import {View, Text, StyleSheet} from 'react-native';
  )
  }
 
- export default function CycleCard({periodDays, daysSinceLastPeriod, cycleDonutPercent}){
+ export default function CycleCard({periodDays, daysSinceLastPeriod, cycleDonutPercent, showTip}){
+   let styleWithTip = showTip ? {} : styles.noTip;
    return (
-        <View style={[styles.card]}>
+        <View style={[styles.card, styleWithTip]}>
```

Comment:
```suggestion
        <View style={[ styles.card, !showTip && styles.noTip ]}>
```
Faster way of doing everything. You don't need line 55 `let styleWithTip = showTip ? {} : styles.noTip;` with this.